openc3-minio/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG OPENC3_DEPENDENCY_REGISTRY=docker.io
-FROM ${OPENC3_DEPENDENCY_REGISTRY}/minio/minio:RELEASE.2022-12-02T19-19-22Z
+FROM ${OPENC3_DEPENDENCY_REGISTRY}/minio/minio:RELEASE.2023-08-09T23-30-22Z
```

Comment:
This breaks the MINIO console with a blank page and `You need to enable JavaScript to run this app.`